### DISCUSSION TITLE
[GitLab]

### DISCUSSION THREAD

**Discussion by thytetgc** [community-scripts/ProxmoxVE#809] [Request script] [answered]
### Application Name

GitLab

### Website

https://docs.gitlab.com/

### Description

GitLab is a Git-based software repository manager with support for wikis, task management, and CI/CD.[1] GitLab is similar to GitHub, but GitLab allows developers to store code on their own servers, rather than on third-party servers. It is free software, distributed under the MIT License.[2] It is available as an Omnibus package

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Lyushen** [comment]
Hi Dear ProxmoxVE Community Script maintainers,

I also want to subscribe under this topic and request a GitLab in LXC container script. The hybrid solution such as Alpine-Docker-GitLab is not ideal. Please add it whenever you have time.

Highly appreciate your work!

**Reply by Porencephaly** [reply]
+1 on this. I would love a GitLab lxc script.

**Comment by tremor021** [comment, marked as answer]
Already exists as Turnkey LXC

![image](https://github.com/user-attachments/assets/3e37130b-93a0-4647-93c4-4e903d8bc0ea)
